spec/examples/running_cluster.rb

```diff
@@ -2,40 +2,62 @@
   require 'timeout'
   require 'json'
   require 'rspec/core'
+  require 'net/http'
```

Comment:
Do we need a Gemfile dep. for this?

Reply:
Nope, its part of stdlib - https://ruby-doc.org/stdlib-2.5.3/libdoc/net/http/rdoc/index.html